### DISCUSSION TITLE
Having some trouble setup a template with "#i"

### DISCUSSION THREAD

**Discussion by Inc0n** [minad/tempel#163] [Q&A]
Whilst the provided examples as shown below works perfectly fine.
```
(inc "#include <" (p (concat (file-name-base (or (buffer-file-name) (buffer-name))) ".h")) ">")
```
I would like to if possible setup another such template with "#i" instead, here is what I have so far
```
(\#i "#include <" (p (concat (file-name-base (or (buffer-file-name) (buffer-name))) ".h")) ">")
```
The slash `\` is needed to escape the '#' for the emacs lisp reader. And thus I thought this should complete the setup. However, even then it's still not usable when I try it in an actual c-mode buffer. Emacs minibuffer showing the message of `user-error: tempel-expand: No completions` is not only mildly infuriating but also defeating. 

It does appear that following test does return `t` so theoretically a template with "#i" is technically possible?
```
(eq (car (with-temp-buffer
        (insert "(\n  \\#i")
        (goto-char (point-max))
        (insert "\n)")
        (goto-char (point-min))
        (read (current-buffer))))
    '\#i)
```
I am at a loss for what possible information I am lacking in possesion to make this work, hence, needing to make a post to seek external expertise for aid. It should go without a saying that any help or suggestions will be greatly appreciated.

**Comment by minad** [comment]
Maybe you want to use tempel-trigger-prefix instead?
